integration/resources/templates/combination/function_with_all_event_types.yaml

```diff
@@ -151,7 +151,7 @@ Outputs:
         - TestSchedule${__StackName__}
         - __StackName__: 
             Fn::Select:
-              - 3
+              - 6
```

Comment:
This is changed to 6 because we are adding prefixes to the stack name?
Seems really prone to errors if we ever change the stack names and miss updating this index for all the tests. 
Can we use a different method like passing in parameters?

Reply:
Updated with passing parameter.